src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -451,6 +453,42 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with ResetMoc
     exportLocations should contain (exportLocation2)
   }
 
+  "getConsignments" should "return all the consignments when consignment filter is passed" in {
```

Comment:
Should all consignment be returned when there is no consignment filter? When there is a consignment filter the results would be filtered based on the arguments to the filter?

Reply:
Actually, there is no check related to the filter in the `consignmentService.getConsignments` method. So, it is just a naming. I have changed it to `return filtered consignments when consignment filter is passed`.